0_RootFS/Rootfs/bundled/conf/profile.d/bb_utils.sh

```diff
@@ -62,15 +62,40 @@ qfind() {
     find "$@" 2>/dev/null
 }
 
-# Function to install license files
-install_license () {
-    if [ $# -eq 0 ]; then
-        echo "Usage: install_license license_file1.txt [license_file2.md, license_file3.rtf, ...]" >&2
-        exit 1
+# Function to automatically install license files at the end of the build
+auto_install_license () {
+    if [[ -z "$(qfind "${prefix}/share/licenses/${SRC_NAME}" -mindepth 1)" ]]; then
+        # There are no licenses already installed, let's find 'em all
+
+        DIR="${WORKSPACE}/srcdir"
+        # Build the list of known names for license files
+        LICENSE_FILENAMES=()
+        for bname in COPYING COPYRIGHT LICENCE LICENSE ; do
```

Comment:
I did not know that `licence` was a valid spelling.  :joy:

Reply:
It's used on this side of the pond :gb: :slightly_smiling_face: And I did find a file with this name in one of our packages, but cannot remember which one now.